### ISSUE TITLE
Header based

### ISSUE THREAD

**Issue by brightokine** [AssassinUKG/Nuclei_public_templates#1]
Thanks for this template.
Can you create another just to test for headers only sqli time based.

Thank you.

**Comment by SujalMeghwal** [comment]
I do not understand what do you mean

**Comment by AssassinUKG** [comment]
Yes, you can, follow the template and replace EXTS for headers to swap out. You can do via regex and Extractors: https://nuclei.projectdiscovery.io/templating-guide/operators/extractors/

**Comment by SujalMeghwal** [comment]
ok i will try to create one but only if i have time for it because i am have some project going on for my school

**Comment by AssassinUKG** [comment]
Np, I am working a full-time Job so same boat. Also, take note, nuclei have changed some things in the latest release. 

Looks like things might change again soon. Will keep updated best as I can. 

Info: https://blog.projectdiscovery.io/nuclei-template-v9-5-0-update/
